src/site/_includes/partials/paged-next.njk

```diff
@@ -0,0 +1,26 @@
+{% from 'macros/post-card.njk' import postCard with context %}
+
+<div class="region">
+  <div class="wrapper auto-grid">
+    {% for element in paged.elements %}
+      {% set data = element.data %}
+
+      {{ postCard({
+        title: data.title,
+        subhead: data.subhead,
+        url: element.url,
+        thumbnail: data.thumbnail or data.hero,
+        alt: data.alt or data.title or '',
+        authors: data.authors,
+        tags: data.tags,
+        date: element.date,
+        updated: data.updated,
+        isDraft: data.draft
```

Comment:
All other elements have the same name as property in element.data. No need to introduce inconsistency here.